netlify/edge-functions/repo-page-card.tsx

```diff
@@ -0,0 +1,229 @@
+import React from "react";
+import { ImageResponse } from "og_edge";
+import type { Config } from "https://edge.netlify.com";
+
+const baseApiUrl = Deno.env.get("NEXT_PUBLIC_API_URL");
+
+type StatsType<T = "stars" | "forks"> = T extends "stars"
+  ? { star_count: number; bucket: string }
+  : { forks_count: number; bucket: string };
+
+function humanizeNumber(num: number | string, type: "comma" | "abbreviation" | null = null) {
```

Comment:
Post release, I will refactor the use of the shared functions that are currently copy pasta.